product-features/farming/README.md

```diff
@@ -0,0 +1,38 @@
+---
+description: Stake your LP tokens and maximize your rewards!
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: false
+---
+
+# 🌾 Farming
+
+Yield Farming on ALEX is an excellent way to earn exciting returns. By farming (staking or locking up your LP tokens), you earn rewards in addition to the earnings from being a liquidity provider. Among these rewards are $ALEX and APower tokens.
```

Comment:
Yield Farming on ALEX is an excellent way to earn exciting returns. By farming (staking or locking up your LP tokens) you earn rewards in addition to the pool fees from being a liquidity provider. Among these rewards are $ALEX and APower tokens.